build.sbt

```
@@ -325,9 +301,9 @@ lazy val nscplugin =
       scalaVersion := libScalaVersion,
       crossScalaVersions := libCrossScalaVersions,
       crossVersion := CrossVersion.full,
-      unmanagedSourceDirectories in Compile ++= Seq(
-        (scalaSource in (nir, Compile)).value,
-        (scalaSource in (util, Compile)).value
+      Compile / unmanagedSourceDirectories ++= Seq(
+        (nir / Compile / scalaSource).value,
+        (util / Compile / scalaSource).value,
```

I accidentally introduced a trailing comma here.
```suggestion
        (util / Compile / scalaSource).value
```